### PR TITLE
Fix Empty ListValue/Struct json encoding

### PR DESCRIPTION
**Pull request by TeBoring**

### CODE REVIEW

**Comment by TeBoring** [comment]
@michaelbausor could you help take a look?

**Comment by TeBoring** [comment]
Fix #5192

**Comment by TeBoring** [comment]
Broken ruby tests should not be related.

File: php/tests/encode_decode_test.php [hunk 1]
```diff
@@ -898,6 +898,13 @@ public function testEncodeTopLevelListValue()
         $this->assertSame("[1.5]", $m->serializeToJsonString());
     }
 
+    public function testEncodeEmptyListValue()
+    {
+        $m = new TestMessage();
+        $m->setRepeatedInt32([]);
```
Comment (michaelbausor) [inline php/tests/encode_decode_test.php]: This doesn't look like it tests ListValue?

Comment (TeBoring) [reply php/tests/encode_decode_test.php]: Oh, I guess I removed the wrong test. Thanks for the catch.

**Comment by TeBoring** [comment]
Thanks for the review!

**Pull request merged**